packages/components/src/internal/components/samples/utils.tsx

```diff
@@ -190,3 +191,15 @@ export const shouldShowButtons = (
 ): boolean => {
     return hideButtons === undefined || hideButtons.indexOf(action) === -1;
 };
+
+export function isSamplesSchema(schemaQuery: SchemaQuery): boolean {
+    const lcSchemaName = schemaQuery?.schemaName?.toLowerCase();
+    if (lcSchemaName === SCHEMAS.SAMPLE_SETS.SCHEMA)
+        return true;
+
+    const lcQueryName = schemaQuery?.queryName?.toLowerCase();
+    if (lcSchemaName === SCHEMAS.EXP_TABLES.SCHEMA && lcQueryName === SCHEMAS.EXP_TABLES.MATERIALS.queryName.toLowerCase())
+        return true;
+
+    return (lcQueryName === SCHEMAS.SAMPLE_MANAGEMENT.SOURCE_SAMPLES.queryName.toLowerCase());
```

Comment:
likely not needed since the SOURCE_SAMPLES query name is unlikely to exist in any other schema namespace, but any reason to have a check on the schema name here as well?